src/pages/index.js

```diff
@@ -22,7 +23,11 @@ const f = foundationPartial(ebiGlobalStyles, ipro, style);
 
 // Main pages
 const Home = loadable({
-  loader: () => import(/* webpackChunkName: "home-page" */ './Home'),
+  loader: () =>
+    import(
+      /* webpackPreload: true */
```

Comment:
I'm wondering what's the proportion of users that enter the website not through the home page but through entry pages directly (following external links, or a Google search). Because for those then this preload might be a waste of resources that might never actually be used.

Reply:
About the `webpackPreload` for the `Home` component, it is about 12% of all visits that go into the homepage, 5% to the search page, and the rest of pages are around 1% or much less. So I do think the unnecessary load in other pages is justifiable. Moreover, thanks to other changes in this PR the main bundled wa reduced to at least half of what it was, so other pages are going to get a booster in performance nonetheless.

Reply:
Thanks for sharing the `lazy.ts` code I'll read it properly and see if we can do something similar!